libsolidity/interface/StandardCompiler.cpp

```diff
@@ -160,6 +160,38 @@ Json::Value collectEVMObject(eth::LinkerObject const& _object, string const* _so
 	return output;
 }
 
+Json::Value compileLLL(Json::Value const& _input)
+{
+	if (!_input["source"].isString())
+		return formatFatalError("JSONError", "Source missing.");
+
+	Json::Value const& settings = _input.get("settings", Json::Value());
+	Json::Value optimizerSettings = settings.get("optimizer", Json::Value());
+	bool optimize = optimizerSettings.get("enabled", Json::Value(false)).asBool();
+
+	vector<string> errorList;
+	Json::Value output;
+	output["contracts"] = Json::objectValue;
+	output["contracts"][""] = Json::objectValue;
```

Comment:
Question is to do we go for
```
{
  "contracts": {
    "": {
      "bytecode": ""
    }
  }
}
```

in order to be "compatible" with the usual, or go with:

```
{
  "bytecode": "..."
}
```

Reply:
Fixed.